camera/rpi-files/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Python environment
+env
+
+# Ignore app logs
+app.log
```

Comment:
maybe move to main gitignore and remove this?

Reply:
I'd rather keep .gitignores separate in different projects. I will move up to camera.

Reply:
Actually, I'd rather keep it as it. `rpi-files` is a separate filesystem for Raspbian, we may ship only this folder to rpi later.